strutil/version.go

```diff
@@ -32,7 +32,7 @@ func max(a, b int) int {
 	return a
 }
 
-//go:generate go run github.com/snapcore/snapd/strutil/chrorder -package=strutil -output=chrorder.go
+//go:generate go run ./chrorder/main.go -package=strutil -output=chrorder.go
```

Comment:
Debian build does not set GOPATH?

Reply:
go run does not take a package, it takes a path to a file